.circleci/config.yml

```diff
@@ -15,6 +15,10 @@ jobs:
       - run:
           name: Check Markdown structure
           command: make lint
+      
+      - run:
```

Comment:
Tthis and `Check Markdown structure` could run in parallel to speed up CI (like we did in happa)

Reply:
They could, but the overhead for spinning up a job is ~10 seconds, while the "Validate front matter" step only takes 10 seconds to execute and "Check Markdown structure" takes 7 seconds. So we would add 20 seconds of machine runtime for zero net gain.